src/main/java/org/myrobotlab/service/SwingGui.java

```diff
@@ -340,8 +340,12 @@ public void run() {
         send(name, "publishState");
 
         if (getName().equals(name) && guiServiceGui == null) {
-          guiServiceGui = (SwingGuiGui) newGui;
-          guiServiceGui.rebuildGraph();
+          try {
+            guiServiceGui = (SwingGuiGui) newGui;
+            guiServiceGui.rebuildGraph();
+          } catch (Exception e) {

```

Comment:
Why do we need this try/catch here.. we didn't have it before?  Don't change this .. please revert.

Reply:
Ok, but I saw an error one time because of this.. will revert